1-js/04-object-basics/04-object-methods/2-check-syntax/solution.md

```diff
@@ -34,4 +34,4 @@ let user = {
 (user.go)() // John
 ```
 
-Please note that brackets around `(user.go)` do nothing here. Usually they setup the order of operations, but here the dot `.` works first anyway, so there's no effect. Only the semicolon thing matters.
+Rețineți că parantezele din jurul `(user.go)` nu fc nimic aici. De obicei setează ordinea operațiunilor, dar aici punctul `.` are oricum precedență, deci nu are niciun efect. Singura chestie care contează este semicolonul.
```

Comment:
'nu fac' in loc de 'nu fc'

Reply:
De asemenea, nu este mai ok 'ordinea operatiilor' in loc de 'ordinea operatiunilor'

Reply:
'Singurul lucru' in loc de 'singura chestie' poate?